cogs/welcome/welcome.py

```diff
@@ -20,6 +24,7 @@
 KEY_TITLE = "title"
 KEY_MESSAGE = "message"
 KEY_IMAGE = "image"
+GREETINGS = []
```

Comment:
I don't see this being used, can we revert this?

Reply:
done with rework